OneCommunityGlobal/HighestGoodNetworkApp
# Kurtis - Fix default cursor appearing on timeEntryForm in the Notes section

# Description
Cursor showed default when hovering over placeholder text within notes when the intangible/tangible time entry modal was open. Thought it was more intuitive and natural if the cursor showed default to indicate the notes entry could be typed within. It would show cursor:text once text was entered prior, but would be default if no text was entered at all.
Fixes # PRIORITY MEDIUM

## Related PRS (if any):
This frontend PR is related to the #886 frontend PR.
…

## Main changes explained:
When adding tangible or intangible time I noticed that when I hovered the cursor over the default placeholder text, the cursor was default. In order to have the box seem more active and accessible from the get go, I implemented a fix that would default the cursor type to "text" which would indicate the textbox in the "Notes" section would be accessible right away one you hovered over the placeholder text since that would be the natural pointer position to interact with it. 

through examining the tinyMCE docs, I saw that I could directly manipulate this through adding                     content_style: 'body { cursor: text !important; }' 
to the init section. Because of the !important, it maintains the highest specificity possible and overrides the default cursor styling that was being imposed by an unknown source. 

…

## How to test:
-I'm still new to Pull requests, so I wasn't sure on the process here. Because of that, you'll need to navigate to the repo that I forked. 
- clone the repo that I forked onto your local environment by using git clone
- do `npm install to update the node_modules
- afterwards, run npm run build 
- followed by npm run start to launch the app on local host to begin testing the modified development client
- log as admin user or volunteer user
- go to dashboard→ add intangible time entry →hover over placeholder text within notes section
- verify function “A” (feel free to include screenshot here)

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/19482363/825cba9c-07bf-4c81-8d61-ba2aad7f6f15

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/19482363/fb1a096e-e0f1-4431-9578-51650a4b2350
